python-base/ubuntu20.04-python3.9/docker/scripts/install_poetry.sh

```diff
@@ -13,3 +13,13 @@ su "${USER}" -c "python /tmp/get-poetry.py --version ${POETRY_VERSION}"
 
 chmod +x "${POETRY_HOME}"/bin/*
 bash /src/docker/scripts/configure_poetry.sh
+
+# Remove useless old vendor packages
+cd /usr/local/poetry/lib/poetry/_vendor/
```

Comment:
Poetry bundles `2.7`, `3.5`, `3.6`, `3.7`, `3.8`, and `3.9` vendored packages by default and they can take a fair bit of space.